server/lib/add_user_database.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request');
```

Comment:
We have decided to use the superagent library for requests, so please can you replace this.